lib/engine-jit/src/artifact.rs

```diff
@@ -48,14 +48,14 @@ impl JITArtifact {
     #[cfg(feature = "compiler")]
     pub fn new(
         jit: &JITEngine,
-        data: &[u8],
+        binary: &[u8],
```

Comment:
It seems this argument is called `data` in all the other engines, we can change it but it should be consistent

`rg -F 'data: &[u8]'` shows the other uses

Reply:
Reverted!